tsconfig.json

```diff
@@ -16,7 +16,7 @@
         "sourceMap": true,
         "strict": true,
         "target": "es5",
-
+        "skipLibCheck": true,
```

Comment:
Why is this needed? :thinking: Doesn't it make the type-checking weaker?

Reply:
There was issues in library types in case when there was no extraneous dependencies in pnpm

Reply:
I don't understand. Can you please elaborate more on it?